servers/physics/broad_phase_octree.cpp

```diff
@@ -31,7 +31,7 @@
 #include "broad_phase_octree.h"
 #include "collision_object_sw.h"
 
-BroadPhaseSW::ID BroadPhaseOctree::create(CollisionObjectSW *p_object, int p_subindex) {
+BroadPhaseSW::ID BroadPhaseOctree::create(CollisionObjectSW *p_object, int p_subindex, const AABB &p_aabb) {
 
 	ID oid = octree.create(p_object, AABB(), p_subindex, false, 1 << p_object->get_type(), 0);
 	return oid;
```

Comment:
Presumably you could do the same for octree? Not that it matters that much if we do get rid of it, but may as well..

Reply:
I've tried but it actually makes things slower to just initialize the aabb, because the way it works right now, having an empty aabb to start with avoids extra checks in `set_static` and things get checked later in `move`.

It should be possible to dig into it and set things properly, but I'm not sure there's going to be a noticeable gain and as you said it's probably not worth it if we start using the bvh instead.